src/components/Modal/SearchModal.js

```diff
@@ -0,0 +1,107 @@
+import React, { useRef, useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import SearchListModal from './SearchListModal';
+import TagModal from './TagModal';
+
+function SearchModal(props) {
```

Comment:
```suggestion
function SearchModal({clickSearch}) {
```
매개변수자리에서 바로 구조분해 가능합니다.

Reply:
그럼 class형 컴포넌트에서처럼 props로 받아온 것들 전부 구조분해할당 해주는 것이 좋나요?🧐

Reply:
회사마다 컨벤션이 다를수는 있습니다. 위코드에서는 객체가 반복적으로 사용되고 있다면 매번 `props. props.` 적는것보다 구조분해하는 것을 컨벤션으로 하고 있습니다!